src/main/java/seedu/address/logic/commands/UpdateProjectCommand.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.project.DeadlineList;
+import seedu.address.model.project.EventList;
+import seedu.address.model.project.ParticipantList;
+import seedu.address.model.project.Project;
+import seedu.address.model.project.ProjectName;
+import seedu.address.model.project.TodoList;
+
+/**
+ * Updates the name of a project in the project list.
+ */
+public class UpdateProjectCommand extends Command {
+
+    public static final String COMMAND_WORD = "updateP";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Updates the name of the project specified "
+            + "by the index number used in the displayed project list. "
+            + "Existing value will be overwritten by the input value.\n"
+            + "Parameters: INDEX (must be positive integer) "
+            + "[" + PREFIX_NAME + "NAME]\n"
```

Comment:
Square brackets are for optional parameters, the parameters here do not seem to be optional during testing.

Reply:
Let's also standardise the naming of parameters with the addP command.